tests/monday_type_tests.py

```diff
@@ -144,4 +145,93 @@ def test_number_type_should_succeed_when_to_primitive_returns_export_string_when
 
     # Assert
     eq_(int_value,'1')
-    eq_(float_value,'1.0')
\ No newline at end of file
+    eq_(float_value,'1.0')
+
+
+def test_date_type_should_succeed_when_to_native_returns_a_datetime_when_passing_a_datevalue_value_with_api_data():
```

Comment:
Everything looks great so far!  

The only thing I ask is to please move these tests into tests/monday_types/date_type_tests.py as per the updated requirements docs.  Thanks!